UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventHighlander.cs

```diff
@@ -35,11 +35,16 @@ public override void OnEventStart()
 
 		private IEnumerator StartRoundTimer()
 		{
+			int Round = GameManager.RoundID;
 			remainingTime = eventTime;
 			while (remainingTime > 0f)
 			{
 				remainingTime -= 1f;
 				yield return WaitFor.Seconds(1f);
+				if (Round != GameManager.RoundID)
+				{
+					yield break; 
```

Comment:
wouldn't this escape the break but still run the next line, which is ending the round?

Reply:
yield break 
I think breaks out of the entire thing

Reply:
okay